ui/src/reducers/playQueue.js

```diff
@@ -81,7 +81,13 @@ export const playQueueReducer = (previousState = initialState, payload) => {
       Object.keys(data).forEach((id) => {
         queue.push(mapToAudioLists(data[id]))
       })
-      return { ...previousState, queue, clear: false, playIndex: undefined }
+      return {
+        ...previousState,
+        queue,
+        clear: false,
+        playIndex: undefined,
+        playing: true,
```

Comment:
Any particular reason why you introduced another value in the store, instead of using the `current.paused` (line 71 above)?

Reply:
I did this to avoid recursive calls being made. Since, the current ```actual play/pause``` depends on ```current.paused```. So, it is flipped everytime I change the status of the song. So, if I listen on to the former i.e. act on the changing of the ```current.paused```, when the togglePlayPause would run again and flip the value again and so forth, triggering a recursion. To avoid this, I introduced another value to listen on to the changes and act accordingly.